pkg/ccl/backupccl/full_cluster_backup_restore_test.go

```diff
@@ -303,6 +303,7 @@ CREATE TABLE data2.foo (a int);
 	})
 
 	t.Run("ensure that jobs are restored", func(t *testing.T) {
+		skip.WithIssue(t, 75216, "flaky under testrace")
```

Comment:
```diff
diff --git i/pkg/ccl/backupccl/full_cluster_backup_restore_test.go w/pkg/ccl/backupccl/full_cluster_backup_restore_test.go
index 5791a2b1be..941f5f51ec 100644
--- i/pkg/ccl/backupccl/full_cluster_backup_restore_test.go
+++ w/pkg/ccl/backupccl/full_cluster_backup_restore_test.go
@@ -48,6 +48,7 @@ func TestFullClusterBackup(t *testing.T) {

        params := base.TestClusterArgs{
                ServerArgs: base.TestServerArgs{
+                      DisableSpanConfigs: true, // TODO(irfansharif): #75060.
                        Knobs: base.TestingKnobs{
                                SpanConfig: &spanconfig.TestingKnobs{
                                        // We compare job progress before and after a restore. Disable
```

Mind using diff instead?

Reply:
And updating the commit message/PR title accordingly. Pretty sure it's fallout from #73876 that I have yet to investigate.

Reply:
Wait I don't see a `DisableSpanConfigs` field under `TestServerArgs`

Reply:
Updated